Makefile

```diff
@@ -121,6 +121,11 @@ dhall_types :
 	ulimit -s 65532 && (ulimit -n 10240 || true) && dune build src/app/dhall_types/dump_dhall_types.exe --profile=dev
 	$(info Build complete)
 
+replayer :
```

Comment:
can you put this in buildkite as well?

Add a step to build this to `buildkite/scripts/build-artifact.sh`

Reply:
Will add.